### DISCUSSION TITLE
How can do AGW with VMWare ESXi?

### DISCUSSION THREAD

**Discussion by velosonetmao** [magma/magma#11159] [Q&A]
The enviorement: 
- VMWare ESXi 6.5 without vCenter, just standalone:

I´m crazying try to get works AGW (1.6.0) with VMWare ESXi, but with no luck. I ´ve installed AGW in VMWare ESXi with two interfaces (eth0 and eth1) both in a separated vswitch. The install ocorrured without problems and AGW can connect at the Orchestration with sucessful. 

Enodeb: Mikrotik Intercell 10 B38+B39

The problem:

I can´t get IMSI connect at the virtualized AGW, even with all services has running fine, without errors.

Could you guys help me with the problem? Anyone was this problem? Or just AGW can´t run in a VMWare enviorement?

**Comment by edaspb** [comment]
I can confirm AGW running on ESXi with full functionality. Do you have s1 connection up? Please share mme and syslogs here. Your issue definitely not related to enviorement.

**Reply by fabioccoelho** [reply]
I already found the problem and fixed it.

Thanks.

**Reply by im-nonedone** [reply]
@fabioccoelho I was wondering what the issue is and how you fixed it? I am running into issues with AGW on vmware, but no issue with AGW on AWS. 

```
10 3.990012769 172.21.20.100 → 172.21.20.101 S1AP/NAS-EPS 266 InitialContextSetupRequest, Attach accept, Activate default EPS bearer context request
   11 4.048728371 172.21.20.101 → 172.21.20.100 S1AP 118 UECapabilityInfoIndication, UECapabilityInformation
   12 4.252068361 172.21.20.100 → 172.21.20.101 SCTP 62 SACK 
   13 4.252921616 172.21.20.101 → 172.21.20.100 S1AP 90 InitialContextSetupFailure [RadioNetwork-cause=invalid-qos-combination]
```

**Comment by lucasgonze** [comment]
That is a valid question ("how did you do it") but the topic can be closed.
